README.md

```diff
@@ -5,7 +5,7 @@
 
 ![screenshot btc nano s](docs/screenshot-api-nanos-btc.png)
 
-The goal of this project is to emulate Ledger Nano S, Nano X and Blue apps on
+The goal of this project is to emulate Ledger Nano S, Nano X, Blue and Stax apps on
```

Comment:
While at it, there is also the Nano S+ which is supported

Reply:
✅